CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5237035)

Reply:
Lässt sich vermutlich relativ einfach bereinigen per `\[Unreleased]`. So ist das ein Shorthand-Link (zum Abschnitt `#Unreleased`), für den es keine Referenz gibt